sails/api/helpers/authenticate-jwt.js

```diff
@@ -0,0 +1,39 @@
+const jwt = require('jsonwebtoken');
+module.exports = {
+
+  friendlyName: 'Authenticate JWT Access Token',
+
+
+  description: 'Returns JWT body data if the given req.headers contains a valid JWT Access Token',
+
```

Comment:
File name doesn't match our upper camel case style (AuthenticateJWT.js or AuthJWT.js, to match AuthController or rename the controller to AuthenticationController).
Line 2: Add an empty line between 1 & 2
Lines 3 to 8: too much empty space